### PR TITLE
[O21] Outer Rim Galaxies patch

### PR DESCRIPTION
**Pull request by Person243546**
## Additions

Adds a compatibility patch for [O21] Outer Rim Galaxies [DevBuild].

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Apparel.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+					<ArmorRating_Blunt>0.9</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="O21_OR_DroidArmorModule"]/statBases/Mass</xpath>
+				<value>
+					<Mass>13</Mass>
+					<Bulk>25</Bulk>
+					<StuffEffectMultiplierArmor>3</StuffEffectMultiplierArmor>
```
Comment (N7Huntsman) [inline Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Apparel.xml:53]: This `StuffEffectMultiplierArmor` is a duplicate node.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
A very quick and dirty first pass pointing out a couple errors. I'll do a more detailed review later on, but since the base mod is still currently a dev build, it's likely too unstable to merge a patch for at the moment.

File: Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Weapons.xml [hunk 2]
```diff
@@ -0,0 +1,661 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>[O21] Outer Rim Galaxies [DevBuild]</li>
```
[the remark this answers is not in the document]
Comment (Person243546) [reply Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Weapons.xml:6]: Should be fixed now

File: Patches/[O21] Outer Rim Galaxies/Chiss/Patch_OuterRim_Chiss_Weapons.xml [hunk 3]
```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>[O21] Outer Rim Galaxies [DevBuild]</li>
```
[the remark this answers is not in the document]
Comment (Person243546) [reply Patches/[O21] Outer Rim Galaxies/Chiss/Patch_OuterRim_Chiss_Weapons.xml:6]: Fixed. All the patches for optional content should have a conditional patch now.

**Pull request closed without merging**